torch/csrc/generic/methods/TensorMath.cwrap

```diff
@@ -1402,7 +1402,9 @@
     long s1 = THTensor_(size)(LIBRARY_STATE ((THPTensor*)$arg4)->cdata, 0);
     long s2 = THTensor_(size)(LIBRARY_STATE ((THPTensor*)$arg5)->cdata, 0);
     THTensor_(resize2d)(LIBRARY_STATE ((THPTensor*)$arg0)->cdata, s1, s2);
+    #ifndef WITH_CUDA
```

Comment:
You have to use `#if !IS_CUDA`. `WITH_CUDA` will always be defined in a CUDA-enabled build (even when compiling CPU tensors).